packages/services/src/infrastructure/api/alex-sdk/alex-sdk.client.ts

```diff
@@ -1,21 +1,44 @@
 import { AlexSDK, TokenInfo } from 'alex-sdk';
+import axios from 'axios';
 
 import { HttpCacheService } from '../../cache/http-cache.service';
 import { HttpCacheTimeMs } from '../../cache/http-cache.utils';
 
+export interface AlexApiTokenPricesResponse {
+  data: AlexApiTokenPrice[];
+}
+export interface AlexApiTokenPrice {
+  contract_id: string;
+  last_price_usd: number;
+}
 export type AlexSdkLatestPrices = Partial<{
   [currency in string]: number;
 }>;
 export type AlexSdkTokenInfo = TokenInfo;
 
 export interface AlexSdkClient {
+  getApiTokenPrices(signal?: AbortSignal): Promise<AlexApiTokenPrice[]>;
   getLatestPrices(): Promise<AlexSdkLatestPrices>;
   getSwappableCurrencies(): Promise<AlexSdkTokenInfo[]>;
 }
 
 const alex = new AlexSDK();
 
 export function createAlexSdkClient(cacheService: HttpCacheService): AlexSdkClient {
+  async function getApiTokenPrices(signal?: AbortSignal) {
+    return await cacheService.fetchWithCache(
+      ['alex-api-client-token-prices'],
+      async () => {
+        const res = await axios.get<AlexApiTokenPricesResponse>(
+          `https://api.alexgo.io/v2/public/token-prices`,
```

Comment:
Maybe this URL can be a constant?

Reply:
I see in most other places we have hardcoded URLs so it's probably fine to leave it as is if they are just used once anyway